app/views/items/show.html.erb

```diff
@@ -4,5 +4,6 @@
 <p>Merchant Name: <%= @merchant.name %></p>
 <p><%= @item.inventory %> in stock</p>
 <p>$<%= @item.price %></p>
+<p>Average Fulfillment Time: <%= @item.average_fulfillment_time %> days</p>
 
 <%= button_to "Add to Cart", carts_path(item_id: @item.id) %>
```

Comment:
On line 9, it needs a conditional statement for only a visitor or regular user to see a link to add this item to their cart. The admin and merchant should not see the "Add to Cart" link.

Reply:
Adds condition. Thank you for catching this, Stella.